src/modules/edit.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable prefer-const */
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable no-else-return */
+/* eslint-disable no-useless-return */
+/* eslint-disable array-callback-return */
+/* eslint-disable consistent-return */
+/* eslint-disable arrow-body-style */
+
+let x = JSON.parse(localStorage.getItem('savedData'));
+export let tasks = [];
+
+if (x) {
+  tasks = x;
+}
+
+export default function edit(e) {
+  const item = e.target;
+
+  if (item.classList.contains('fa-trash-can')) {
+    const removeParent = item.parentElement.parentElement;
+    removeParent.remove();
+
+    const removeTaskFromArray = item.parentElement.previousElementSibling.children[0].getAttribute('name');
+
+    tasks = tasks
+      .filter((newTask) => {
+        if (newTask.index !== +removeTaskFromArray) {
+          return true;
+        }
+      })
+      .sort((a, b) => {
+        if (a.index !== 1) {
+          a.index = 1;
+        }
+        return a.index - b.index;
+      })
+      .sort((task, newTask) => {
+        if (task.index + 1 === newTask.index) {
+          return;
+        } else {
+          newTask.index = task.index + 1;
+        }
+      });
+
+    localStorage.setItem('savedData', JSON.stringify(tasks));
+    console.log(tasks);
```

Comment:
- [ ] I would recommend removing the `console.log` statement since printing any information is not a good idea for security purposes, Please remove it.

Reply:
@Dipeshtwis  done